ENG-3686 Coordination ticket for the rebase of 6.3.2/6.5 under operator 7.0

entgigi opened the pull request.

entgigi committed:
ENG-3686 added configurations to customize db images

ENG-3686 fix typo

udpate generate-manifests.sh to properly handle specific controller coordinator banches

entgigi commented:
to use controller-coordinator from 7.0.2-ENG-3686-PR-103+BB-release+2F+7.0 i did a tag (v7.0.2-ENG-3686) on controller-coordinator

ffleandro reviewed (approved):
LGTM, but I think @firegloves should also review this since he has more experience with this project.

user-54dbb57a committed:
Add pipeline 1.4.0

entgigi committed:
ENG-3686 fixed wrong bundle version

ENG-3686 fix change coordinator version to snapshot

ENG-3686 fix wrong sha values for enatndo-de-app

ENG-3686 aligned values.yaml to the 6.5.1 release excel file, fixed rebase conflicts

ENG-3686 aligned values.yaml to the 6.5.1 release excel file

entgigi committed:
ENG-3686 add generated manifest for version 6.5.1 to use to test
